service/src/main/java/com/epam/esm/service/impl/CertificateServiceImpl.java

```diff
@@ -121,10 +123,10 @@ public List<CertificateDto> showByTagName(String tagName) {
     public List<CertificateDto> showByPartWord(String partWord) {
```

Comment:
search by part of name/description (can be implemented, using DB function call) - that means - if any of columns (name or description) contains a part of a word  - you are to select this GiftCertificate - incorrect current logic

Reply:
Okey, probably, I got you. Try to fix.